ci/build.sh

```diff
@@ -12,8 +12,8 @@ case $COMPONENT in
   client)
     cd client
     npm install
-    npm run build
-    npm run bundle
+    # Use production config, since we want to use these bundles for deploys
+    npm_config_configpath="config/prod/*.purs" npm run build
```

Comment:
I don't know the exact rules of npm env vars. Am I correct in that:
* `npm_config_configpath` is equivalent to overriding the `config.configpath` settings in package.json
* `npm_package_config_configpath` is equivalent to referencing `config.configpath` within the current package.json.

Reply:
As I understand it, `npm_config_*` is for configuring the npm client itself, and is only influenced by npmrc files, not anything inside package.json. `npm_package_config_*` is indeed equivalent to referencing `config.configpath` inside `package.json`, unless that configuration option has been overridden by an environment variable, as it has here.